gulpfile.js

```diff
@@ -56,6 +56,18 @@ function watch() {
   gulp.watch(copyFileSources, copy);
 }
 
+var typedoc = require('gulp-typedoc');
```

Comment:
1. Prefer block scoping keywords. In this case, you want `const` since you don't mutate it.
2. You should move this `require()` call to the top of the file with the other `require`s. This mimics the hoisting behavior of ES modules, even if we're not using them in this file.